packages/templates/src/raw/album-releases/metaData.js

```diff
@@ -31,10 +31,26 @@ export default {
     _x('Blue', 'template keyword', 'web-stories'),
   ],
   colors: [
-    { label: _x('Hot Orange', 'color', 'web-stories'), color: '#f7722f' },
-    { label: _x('Peach Bisque', 'color', 'web-stories'), color: '#fee2c6' },
-    { label: _x('Champagne Yellow', 'color', 'web-stories'), color: '#f4e6b0' },
-    { label: _x('Cola Brown', 'color', 'web-stories'), color: '#3d3225' },
+    {
+      label: _x('Hot Orange', 'color', 'web-stories'),
+      color: '#f7722f',
+      family: _x('Orange', 'color', 'web-stories'),
+    },
+    {
+      label: _x('Peach Bisque', 'color', 'web-stories'),
+      color: '#fee2c6',
+      family: _x('Red', 'color', 'web-stories'),
```

Comment:
Just curious, how did you decide the family? I would think a `Peach` would fall under `Orange`🤔

Reply:
I'm assuming there's a lot scenarios where some of these could go several ways, and there's a lot of colors here to go through.

imo if they're not glaringly wrong, I'm fine with them. the only way I'd know how to do this is eyeballing it, so I'm assuming there's a lot of subjectivity involved here

Reply:
Max is spot on, some of these are subjective. This one is probably better off orange, i will update it.

Reply:
That was my assumption, I was just curious if there was some magic way I didn't know of. Going through these by hand must have been brutal. 💪 Thanks for doing it!